### PR TITLE
Add graphql_connection_amount_requested filter

### PR DESCRIPTION
**Pull request by esamattis**
### Your checklist for this pull request
Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.

🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are making a pull request against the **develop branch** (left side). Also you should start *your branch* off *our develop*.
- [x] Make sure you are requesting to pull request from a **topic/feature/bugfix branch** (right side). Don't pull request from your master!

What does this implement/fix? Explain your changes.
---------------------------------------------------

Add ability for extension plugins to modify the connection pages sizes.

wp-graphql-offset-pagination would use this 

https://github.com/valu-digital/wp-graphql-offset-pagination/pull/5

### CODE REVIEW

File: src/Data/Connection/AbstractConnectionResolver.php [hunk 1]
```diff
@@ -409,7 +409,13 @@ public function get_amount_requested() {
 			}
 		}
 
-		return max( 0, $amount_requested );
+        /**
```
Comment (jasonbahl) [inline src/Data/Connection/AbstractConnectionResolver.php:412]: looks like spacing is a bit funky between the comment and the code?

Comment (esamattis) [reply src/Data/Connection/AbstractConnectionResolver.php]: Converted the spaces to tabs. If that's what you mean?

Comment (esamattis) [reply src/Data/Connection/AbstractConnectionResolver.php]: ah there's code sniffer workflow

**Comment by esamattis** [comment]
The schema linter error seems to be unrelated 
image

**Comment by jasonbahl** [comment]
@esamattis yes, the Schema Linter _is_ unrelated. Been breaking for a while 🤦‍♂️

**Pull request merged**